src/modules/helpchan.ts

```diff
@@ -81,13 +84,68 @@ export class HelpChanModule extends Module {
 		return `${this.CHANNEL_PREFIX}${decidedChannel}`;
 	}
 
+	private getOngoingChannels() {
+		return this.client.channels.cache
+			.filter(
+				channel =>
+					(channel as TextChannel).parentID === categories.ongoing,
```

Comment:
Are you sure casting this is safe?

Reply:
Yes, since all channels inside the ongoing category will be text channels.